src/qt/guiutil.cpp

```diff
@@ -673,10 +677,10 @@ bool SetStartOnSystemStartup(bool fAutoStart)
         // Write a bitcoin.desktop file to the autostart directory:
         optionFile << "[Desktop Entry]\n";
         optionFile << "Type=Application\n";
-        if (chain == CBaseChainParams::MAIN)
-            optionFile << "Name=Bitcoin\n";
+        if (chain == CBaseChainParams::LIQUID1)
+            optionFile << "Name=Liquid\n";
```

Comment:
Based on the other changes in this PR, I would expect this to be adding LIQUID1, not replacing MAIN. (But I don't really know what I'm looking for.) Is it correct as written here?

Reply:
Ah yeah sure, that makes sense. Leaving MAIN in place is the goal as it reduces diff. Good catch.